app-config.yaml

```diff
@@ -23,6 +23,9 @@ app:
             title: '#backstage'
 
 backend:
+  # Used for enabling authorization, secret is shared by all backend plugins
+  authorization:
+    secret: ${BACKEND_SECRET}
```

Comment:
Should we add this to the `create-app` template? Also might want to update `backend-common/config.d.ts`

Reply:
I'm leaning towards making it optional, so that only orgs that want to add authentication middleware or use permissions need to set it up

Reply:
Maybe a commented out addition to app config in create app makes sense for discoverability.

Reply:
would be good to add it for secure by default.

maybe include guidance on the secret,

```suggestion
    secret: ${BACKEND_SECRET} # e.g. use openssl rand -base64 32
```